src/content/docs/reference/configuration.mdx

```diff
@@ -425,6 +425,12 @@ The attribute position style in HTMLish languages.
 
 > Default: `"auto"`
 
+### `formatter.bracketSpacing`
```

Comment:
This option is already available in the current release of Biome, but was not documented yet.